WoWsShipBuilder.ViewModels/Other/ShipComparisonDataSectionExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using WoWsShipBuilder.Core.Localization;
+
+namespace WoWsShipBuilder.ViewModels.Other;
+
+public static class ShipComparisonDataSectionExtensions
+{
+    public static string ShipComparisonDataSectionToString(this ShipComparisonDataSections dataSection)
+    {
+        return dataSection switch
+        {
+            ShipComparisonDataSections.General => "General",
+            ShipComparisonDataSections.MainBattery => "MainBattery",
+            ShipComparisonDataSections.He => "He",
+            ShipComparisonDataSections.Ap => "Ap",
+            ShipComparisonDataSections.Sap => "Sap",
+            ShipComparisonDataSections.Torpedo => "Torpedo",
+            ShipComparisonDataSections.SecondaryBattery => "SecondaryBattery",
+            ShipComparisonDataSections.SecondaryBatteryShells => "SecondaryBatteryShells",
+            ShipComparisonDataSections.AntiAir => "AntiAir",
+            ShipComparisonDataSections.Asw => "Asw",
+            ShipComparisonDataSections.AirStrike => "AirStrike",
+            ShipComparisonDataSections.Maneuverability => "Maneuverability",
+            ShipComparisonDataSections.Concealment => "Concealment",
+            ShipComparisonDataSections.Survivability => "Survivability",
+            ShipComparisonDataSections.RocketPlanes => "RocketPlanes",
+            ShipComparisonDataSections.Rockets => "Rockets",
+            ShipComparisonDataSections.TorpedoBombers => "TorpedoBombers",
+            ShipComparisonDataSections.AerialTorpedoes => "AerialTorpedoes",
+            ShipComparisonDataSections.Bombers => "Bombers",
+            ShipComparisonDataSections.Bombs => "Bombs",
+            ShipComparisonDataSections.Sonar => "Sonar",
+            _ => dataSection.ToString(),
+        };
+    }
+
+    public static string LocalizeShipComparisonDataSection(this ShipComparisonDataSections dataSection, ILocalizer localizer)
```

Comment:
again, shorten the name. The extension method already implies it is related to the ship comparison data section, maybe just call it Localize